aws-common-runtime/CMakeLists.txt

```diff
@@ -27,11 +27,11 @@ if (UNIX AND NOT APPLE)
 endif()
 
 set(AWS_C_IO_URL "https://github.com/awslabs/aws-c-io.git")
-set(AWS_C_IO_SHA "v0.2.2")
+set(AWS_C_IO_SHA "c109ffe3156f759723e0609b46182201c65a64e4")
```

Comment:
why are we not running off of tags here? I'm assuming it's just because we haven't cut them yet.

Reply:
Correct, I need to update IO, then I'll get back on a version tag